components/accounts/statement/statement.vue

```diff
@@ -118,6 +118,16 @@ export default class AccountStatement extends mixins(
         return this.monthsFilter(this.account!.creationTimestamp)
     }
 
+    // TODO: Resolve to useBase Composable
+    formatDate(val) {
+        const dateTime = DateTime.fromJSDate(val)
+        if (dateTime.hasSame(DateTime.now(), 'year')) {
+            return dateTime.toFormat('d MMMM')
+        }
+
+        return dateTime.toFormat('d MMMM yyyy')
+    }
```

Comment:
Yep was gonna mention that maybe this should go to a utils file or base composable as you mentioned

Reply:
correct!